tests/unit-tests/graphpartitioning/array_readwriter/csv.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import pandas as pd
+import pyarrow
+import pyarrow.csv
+
+from .registry import register_array_parser
+
+
+@register_array_parser("csv")
+class CSVArrayParser(object):
```

Comment:
Any doc about this CSVArrayParser? Why we need this?

Reply:
This is used to create test datasets on the fly. This is also a apart of integration testing for the end-to-end pipeline.

Reply:
The original code, as existing in the DGL repo today, was created by Minjie when the end-to-end testing pipeline was created around the last 3 months of last year.